src/KafkaFlow/Configuration/TopicConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+namespace KafkaFlow.Configuration
+{
+    /// <summary>
+    /// Represents a Topic configuration
+    /// </summary>
+    public class TopicConfiguration
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TopicConfiguration"/> class.
+        /// </summary>
+        /// <param name="name">The topic name</param>
+        /// <param name="numberOfPartitions">The number of partitions for the topic</param>
+        /// <param name="replicationFactor">Replication factor for the topic</param>
+        public TopicConfiguration(string name, int numberOfPartitions, short replicationFactor)
```

Comment:
I think we can rename the variables to "partitions" and "replicas", what do you think?

Reply:
properties too

Reply:
Renamed.